src/energiapy/graph/graph.py

```diff
@@ -25,17 +24,18 @@ def distance(u_x: float, u_y: float, v_x: float, v_y: float, method: str = 'eucl
     Returns:
         float: _description_
     """
-    if 'rectilinear':
-        distance = abs(u_x-v_x) + abs(u_y-v_y)
+    if method == 'rectilinear':
+        return abs(u_x - v_x) + abs(u_y - v_y)
+    elif method == 'chebyshev':
+        return max(abs(u_x - v_x), abs(u_y - v_y))
+    elif method == 'euclidean':
+        return math.sqrt((u_x - v_x) ** 2 + (u_y - v_y) ** 2)
+    else:
+        raise ValueError('Unknown distance method')
 
-    elif 'chebyshev':
-        distance = max(abs(u_x-v_x) + abs(u_y-v_y))
-    elif 'euclidean':
-        distance = math.sqrt((u_x-v_x)**2 + (u_y-v_y)**2)
-    return distance
 
-
-def make_graph(n_sources: int, n_sinks: int, n_facilities: int, method: str = 'euclidean') -> Tuple[dict, dict, dict]:
+def make_graph(n_sources: int, n_sinks: int, n_facilities: int, method: str = 'euclidean', random_seed: int = 12345) -> \
```

Comment:
changed method definition to include default seed, and changed return type sig to match function